modules/govuk_scripts/files/usr/local/bin/govuk_app_console

```diff
@@ -9,19 +9,19 @@
 
 set -eu
 
-if [ "$#" -ne "1" ] || [ "$1" = "-h" ]; then
-  echo "Usage: $(basename "$0") app_name" >&2
+if [ "$#" -gt 0 ] || [ "$1" = "-h" ]; then
+  echo "Usage: $(basename "$0") app_name [--sandbox]" >&2
   exit 1
 fi
 
 cd /var/apps/$1
 
 if [ -f bin/rails ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./bin/rails console
+  sudo -u deploy govuk_setenv $1 bundle exec ./bin/rails console $2
 elif [ -f script/rails ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./script/rails console
+  sudo -u deploy govuk_setenv $1 bundle exec ./script/rails console $2
 elif [ -f console ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./console
+  sudo -u deploy govuk_setenv $1 bundle exec ./console $2
```

Comment:
I didn't know of this before, but it looks to be a Rails thing. This part of the script seems to be used for Rummager (I didn't see any other apps when I looked).

I'm guessing it won't work for Rummager, so maybe don't pass in the option, or show a message, or both?

Reply:
Correct, rummager is a Sinatra app so it's not likely to support this option (which I'd not heard of either).

I don't *think* we have any more Sinatra apps?  So yeah, should be OK to just error out if this option is provided in a Sinatra context.  We probably don't want to warn and continue as someone might do something they later regret, thinking it'll be rolled back.